CONTRIBUTING.md

```diff
@@ -6,25 +6,30 @@ There are multiple ways to help! We rely on users around the world to help keep
 
 Please read our [Code of Conduct](CODE_OF_CONDUCT.md) before contributing. Everyone interacting with this project (or associated projects) is expected to abide by its terms.
 
+## General note
+
+The definitions for this project are housed in a submodule. Please make sure to run `git clone --recursive git@github.com/holidays/holidays`
```

Comment:
Okay, so I've added `--init` to the `make update_defs` target and it seems to give us what we want. If you do the initial `git clone --recursive` then you will get the submodule as expected, but if you switch to another branch then you have to do the `make update_defs` with the new `--init` in order to get the latest. I tested this with a brand-new clone and I also verified that running with `--init` more than once doesn't cause issues.

Think we are good! Can you confirm @ttwo32?

Addressed in https://github.com/holidays/holidays/pull/222/commits/c52b13c553ac9ca2c0a0d8e0dbe9ccb1481ae1ce